commons/tools/ssl-certificates.md

```diff
@@ -10,11 +10,19 @@ tags:
 - encrypt
 ---
 
-TLS certificates are necessary to enable secure HTTPS connections between a browser and you application. These certificates can be bought online, or directly via Clever Cloud (Domain Validation or Wildcard SSL). In that case, we manage the whole process for you.
+TLS certificates are necessary to enable secure HTTPS connections between a browser and your application. These certificates can be bought online, or generated by Clever Cloud. In that case, we manage the whole process for you.  
 
-If you already have your certificates, you can upload them on Clever Cloud. 
+There is two ways to get HTTPS for your apps:
 
-**About Let's Encrypt**: The service is currently in beta. You can ask for a Let's Encrypt certificate via the support. An integrated user interface is coming soon.
+1. **Let's Encrypt**: This is the quick and easy way to get a TLS certificate on Clever Cloud.
+2. **Your own certificates**: if your already generate have a certificate, you can upload it securely.
+
+## Getting a Let's Encrypt certificate
+
+We automatically generate certificates when you add a domain name to your app.
+It's all you have to do, our internal certificate generator will create a TLS certificate for a domain or a sub-domain.
+
+To get wildcard or Domain Validation certificates, please ask to the support to buy one of these.
```

Comment:
s/To get wildcard or Domain Validation certificates, please ask to the support to buy one of these./We do not support wildcard Let's Encrypt certificates at the moment. If you need a wildcard certificate, you can either generate a Let's Encrypt certificate yourself or buy one (we can resell one to you, in which case we will deal with the csr generation ourselves).

If you want to mention extended validation certificates (not domain validation), I guess you should add a new section.